code/src/components/TodoCountComplete/TodoCounterComplete.css

```diff
@@ -0,0 +1,7 @@
+.complete-text{
```

Comment:
check your Prettier because sometimes you have 2 and sometimes 4 spaces indentation. It should be always 2.